src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs

```diff
@@ -69,6 +73,23 @@ public void ConfigureServices(IServiceCollection services)
                     }
                 });
 
+            services.AddHealthChecks()
+                .AddCheck(
+                    name: "self",
+                    check: () => HealthCheckResult.Healthy(),
+                    tags: new[] { HealthCheckTags.Live })
```

Comment:
I'm not sure specifying the argument names is adding much here. Personally, I prefer it without them but see what you think.

Reply:
I find them helpful as they add more description, however most IDE's do that for you so there's really no need for them. I'll get rid of them.